lte/gateway/python/integ_tests/defs.mk

```diff
@@ -278,7 +278,30 @@ s1aptests/test_restore_config_after_non_sanity.py
 
 # Add the s1aptester integration tests with federation gateway
 FEDERATED_TESTS = federated_tests/s1aptests/test_attach_detach.py \
```

Comment:
Have you tried to run these tests on github actions? How long did it take? Did they run?

Reply:
Tests run is here https://github.com/orion-belt/magma/runs/7745520159?check_suite_focus=true and it takes ~11 min.